Expecto/Expect.fs

```diff
@@ -524,6 +525,30 @@ let sequenceStarts (subject : _ seq) (prefix : _ seq) message =
                       message i (pi.Current)
     i <- i + 1
 
+/// Expect the sequence `actual` to contains elements from sequence `expected` in the right order.
+let sequenceContainsOrder (actual: seq<'t>) (expected: seq<'t>) msg =
+  let el = System.Collections.Generic.Queue<'t> expected
+  use ae = actual.GetEnumerator()
+  let nl = Environment.NewLine
+  let al = ResizeArray<'t>()
+  let missingFail expected iter missing = 
+    Tests.failtestf "%s. Remainder of expected enumerable:%s%A%sWent through actual enumerable (%i items):%A%s" msg nl expected nl iter missing nl
```

Comment:
I don't want `%A` printing of sequences, since that doesn't print them all.

Further, I'd like two newlines between the remainder's output and the next expecto message.

Reply:
Ok :)